docs/self-managed/concepts/exporters.md

```diff
@@ -59,7 +59,7 @@ During the loading phase, the configuration for each exporter is validated, such
 - An exporter instance throws an exception in its `Exporter#configure` method.
 
 The last point is there to provide individual exporters to perform lightweight
-validation of their configuration (e.g. fail if missing arguments).
+validation of their configuration (e.g. fail if missing arguments). For validation call, to configure the exporter context we pass a null partition id, which gets replaced with the correct one at runtime.
```

Comment:
:wrench: 
```suggestion
validation of their configuration (e.g. fail if missing arguments). For this validation call, the given context will report a partition ID (via `Context#getPartitionId()`) with a pseudo-null value, i.e. `Context#NULL_PARTITION_VALUE`). However, at runtime, it will report the appropriate partition ID on which the exporter is running.
```